samples/sensor/vl53l0x/src/main.c

```diff
@@ -27,13 +27,13 @@ void main(void)
 			return;
 		}
 
-		ret = sensor_channel_get(dev, SENSOR_CHAN_PROX, &prox_value);
-		printk("prox is %d\n", prox_value.val1);
+		ret = sensor_channel_get(dev, SENSOR_CHAN_PROX, &value);
+		printk("prox is %d\n", value.val1);
```

Comment:
Is it normal to keep printk here ? Wouldn't be better to use printf everywhere ? 
(Not sure what should be the usage of printk in Zephyr...)

Reply:
(Some) other samples have it like that too. And generally, the idea of making changes is "make minimal changes required to achieve the needed effect, split unrelated changes to separate commits". So, if there would be idea to "make samples consistently use printf(), not combination of printk/printf", that should be a separate commit (and PR, apparently). I don't think it's a priority for now - it works, and we have bigger things in queue ;-).

> You missed to include stdio.h for printf

Fixed.

Thanks!

Reply:
ok, thanks for explanation